config/supported_ssg.yaml

```
@@ -0,0 +1,425 @@
+    version: 0.1.28
+    profiles:
+      C2S:
+        policy_version: 
+      common:
+        policy_version: not versioned
+      CS2:
+        policy_version: not versioned
+      CSCF-RHEL6-MLS:
+        policy_version: 
+      nist-cl-il-al:
+        policy_version: 
+      pci-dss:
+        policy_version: 
+      rht-ccp:
+        policy_version: 
+      server:
+        policy_version: not versioned
+      standard:
+        policy_version: not versioned
+      stig-rhel6-server-upstream:
+        policy_version: 
+      usgcb-rhel6-server:
+        policy_version: 
+  RHEL-7.1:
+    package: scap-security-guide-0.1.19-2.el7
+    brew_uri: http://download.eng.bos.redhat.com/brewroot/vol/rhel-7/packages/scap-security-guide/0.1.19/2.el7/noarch/scap-security-guide-0.1.19-2.el7.noarch.rpm
+    version: 0.1.19
+    profiles:
+      rht-ccp:
+        policy_version: not versioned
+  RHEL-7.2:
+    package: scap-security-guide-0.1.25-3.el7
+    brew_uri: http://download.eng.bos.redhat.com/brewroot/vol/rhel-7/packages/scap-security-guide/0.1.25/3.el7/noarch/scap-security-guide-0.1.25-3.el7.noarch.rpm
+    version: 0.1.25
+    profiles:
+      common:
+        policy_version: not versioned
+      pci-dss:
+        policy_version: draft
+      rht-ccp:
+        policy_version: not versioned
+      standard:
+        policy_version: not versioned
+      stig-rhel7-server-upstream:
+        policy_version: draft
+  RHEL-7.3:
+    package: scap-security-guide-0.1.30-5.el7_3
+    brew_uri: http://download.eng.bos.redhat.com/brewroot/vol/rhel-7/packages/scap-security-guide/0.1.30/5.el7_3/noarch/scap-security-guide-0.1.30-5.el7_3.noarch.rpm
+    version: 0.1.30
+    profiles:
+      C2S:
+        policy_version: 
+      cjis-rhel7-server:
+        policy_version: 
+      common:
+        policy_version: not versioned
+      nist-cl-il-al:
+        policy_version: 
+      ospp-rhel7-server:
+        policy_version: 
+      pci-dss:
+        policy_version: '3.1'
+      rht-ccp:
+        policy_version: not versioned
+      standard:
+        policy_version: not versioned
+      stig-rhel7-server-gui-upstream:
+        policy_version: 
+      stig-rhel7-server-upstream:
+        policy_version: 
+      stig-rhel7-workstation-upstream:
+        policy_version: 
+  RHEL-7.4:
+    package: scap-security-guide-0.1.33-6.el7_4
+    brew_uri: http://download.eng.bos.redhat.com/brewroot/vol/rhel-7/packages/scap-security-guide/0.1.33/6.el7_4/noarch/scap-security-guide-0.1.33-6.el7_4.noarch.rpm
+    version: 0.1.33
+    profiles:
+      C2S:
+        policy_version: 
+      cjis-rhel7-server:
+        policy_version: 
+      common:
+        policy_version: not versioned
+      docker-host:
+        policy_version: 
+      nist-800-171-cui:
+        policy_version: 
+      ospp-rhel7:
+        policy_version: 
```

We need some way to track the change in the ref_id, like:
```suggestion
      ospp-rhel7:
        old: ospp-rhel7-server
        policy_version: 
```

---

Added `old_names`, an array. Includes also those profiles that were merged into it.